src/main/java/com/garrytrue/repository/StudentRepository.java

```diff
@@ -0,0 +1,33 @@
+package com.garrytrue.repository;
+
+
+import com.garrytrue.model.Student;
+import com.garrytrue.storage.IStorage;
+
+/**
+ * Created by garrytrue on 12.03.17.
+ */
+public class StudentRepository implements IRepository<Student> {
```

Comment:
Rename this implementation to HashMapRepository. And it must be Generic too like CRUDRepository<T>. No need to be Students only. Later if you have some code that is working not only with Student but with other entities you will create a separate impls for each entity.